sympy/calculus/hamilton.py

```diff
@@ -0,0 +1,75 @@
+"""
+This module implements a method to find
+Hamilton's Equations of motion for given Hamiltonian.
+"""
+from sympy import Function, sympify, diff, Eq, S, Symbol, Derivative
+from sympy.core.compatibility import (iterable, range)
+
+
+def hamilton_equations(H, coords, momenta, time):
+    r"""
+    Find the Euler-Lagrange equations [1]_ for a given Lagrangian.
```

Comment:
Should this read "Hamilton's"?

Reply:
I will definitely have to change the comment. But I kept it Hamilton instead of Hamilton's because in the other function it was Euler instead of Euler's. So just for consistency, tell me if I should make it Hamilton's.

Reply:
I think non-possessive is ok; in this case the name acts as an adjective as in "Hamiltonian mechanics".

Reply:
Would "Return the equations of the Hamiltonian..." work?

Reply:
or "Hamiltonian equations of motion".